src/event-bus/create-user-handler.ts

```diff
@@ -0,0 +1,11 @@
+import { handlerFor } from '@node-ts/bus-core';
+import { session } from '../neo4j';
+import { CreateUserCommand } from './create-user-command';
+
+export const createUserHandler = handlerFor(CreateUserCommand, async (command) => {
+  await session.run(`CREATE (user:USER {name: $name, email: $email, password: $password})`, {
+    name: command.userCreationParams.name,
+    email: command.userCreationParams.email,
+    password: command.userCreationParams.password,
+  });
```

Comment:
aqui da pra passar o objeto inteiro como parâmetro
```suggestion
  await session.run(`CREATE (user:USER {name: $input.name, email: $input.email, password: $input.password})`, {
    input: command.userCreationParams,
  });
```